packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/customTemplates.test.ts

```diff
@@ -38,40 +39,39 @@ describe('Custom Templates Create', () => {
   let openTextDocumentStub: SinonStub;
   let sendCommandEventStub: SinonStub;
   let sendExceptionStub: SinonStub;
-  let getConfigValue: SinonStub;
+  let getTemplatesDirectoryStub: SinonStub;
 
   beforeEach(() => {
-    showInputBoxStub = stub(vscode.window, 'showInputBox');
-    quickPickStub = stub(vscode.window, 'showQuickPick');
-    appendLineStub = stub(channelService, 'appendLine');
-    showSuccessfulExecutionStub = stub(
+    sandbox = createSandbox();
```

Comment:
late to the game comment: sandbox only needs to be created once so this should be done either outside of the describe or just inside to the top level describe.